chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/ErrorDataSetRenderer.java

```diff
@@ -132,7 +133,7 @@ public Marker getMarker() {
     }
 
     @Override
-    public void render(final GraphicsContext gc, final Chart chart, final int dataSetOffset,
+    public List<DataSet> render(final GraphicsContext gc, final Chart chart, final int dataSetOffset,
```

Comment:
Codacy found an issue: [The method 'render(GraphicsContext, Chart, int, ObservableList)' has an NPath complexity of 81984](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5749297)